### PR TITLE
Added accessing functions for JsonValue

### PR DESCRIPTION
**Pull request by Anonymus-Player**

### CODE REVIEW

File: cpp/Json.hpp [hunk 1]
```diff
@@ -350,6 +368,49 @@ namespace hsd {
...
+
+    template <typename CharT>
+    JsonValue& JsonValue::access(const basic_string_view<CharT>& key)
+    {
+        auto& _map = try_as<JsonObject<CharT>>(JsonValueType::Object).unwrap().values();
+        return *_map.at(key).unwrap();
+    }
+
+    template <typename CharT, usize N>
+    JsonValue& JsonValue::access(const CharT (&key)[N])
```
Comment (dekrain) [inline cpp/Json.hpp:403]: I'm pretty sure we should rather move this into `basic_string_view`, rather than do it manually here

Comment (Anonymus-Player) [reply cpp/Json.hpp]: it's moved into the string view and i've added literals

**Comment by dekrain** [comment]
Overall fine, but the functions defined at the namespace scope (for example `JsonValue::access`) should be declared with the `inline` specifier.

**Comment by Anonymus-Player** [comment]
> Overall fine, but the functions defined at the namespace scope (for example `JsonValue::access`) should be declared with the `inline` specifier.

This commit resolve it

**Pull request merged**